### PR TITLE
Add Super Robot Taisen UX

### PR DESCRIPTION
**Pull request by Digijose**
Adds Super Robot Taisen UX.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by Leo626** [comment]
Also that boxart is really low quality. I can't seem to find a higher quality one so I guess that one will have to do.

File: games/super-robot-taisen-ux/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Super Robot Taisen UX"
+description = "Super Robot Wars UX is a tactical role-playing game for the Nintendo 3DS developed by Banpresto and published by Bandai Namco Games. It is the first Super Robot Wars (SRW) game for the 3DS and was released in Japan on March 14, 2013."
```
Comment (Leo626) [inline games/super-robot-taisen-ux/game.dat]: No need for release date in the description (it's already in the game.dat)

File: games/super-robot-taisen-ux/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+needs_system_files = false
+needs_shared_font = true
+
+[[releases]]
+title = "0004000000078A00"
+region = "jpn"
+release_date = "2013-03-13"
```
Comment (Leo626) [inline games/super-robot-taisen-ux/game.dat]: The release date is actually on the 14th of March, 2013.

File: games/super-robot-taisen-ux/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+[[testcases]]
+title = "0004000000078A00"
+compatibility = "1"
+date = "2017-10-14"
+version = "HEAD-e1bb198"
+cpu = "Intel Core i7-4820K"
+gpu = "Geforce GTX 980 Ti"
```
Comment (Leo626) [inline games/super-robot-taisen-ux/game.dat]: `Nividia`

**Comment by Digijose** [comment]
I don't know how to create the wiki page, i made the screenshots with steam and i don't want to install more software, so i won't update anything of this (beacause i don't know how or even if i can do it).
Thanks for the emu, and if someone read this, Super Robot Taisen UX and BX are playable on citra, yay!

**Pull request closed without merging**